<!Doctype html>
<html class="no-js" lang="">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <title>OPEC Labs</title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <link rel="icon" href="img/header-logo.png">
        <!-- Place favicon.ico in the root directory -->
		
		<!-- Google Fonts -->
		<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,500,500i,700,700i" rel="stylesheet">

		<link rel="stylesheet" href="css/bootstrap.min.css">
		<link rel="stylesheet" href="css/slicknav.min.css">
		<link rel="stylesheet" href="css/bootstrap-theme.min.css">
		<link rel="stylesheet" href="css/owl.carousel.min.css">
		<link rel="stylesheet" href="css/owl.theme.default.min.css">
		<link rel="stylesheet" href="css/animate.min.css">
		<link rel="stylesheet" href="css/animate-text.css">
		<link rel="stylesheet" href="css/magnific-popup.css">
        <link rel="stylesheet" href="css/font-awesome.min.css">
		
		<!-- Ruby CSS -->
        <link rel="stylesheet" href="css/normalize.css">
        <link rel="stylesheet" href="style.css">
		<link rel="stylesheet" href="css/responsive.css">
		
		<!-- Color CSS -->
		<!-- <link rel="stylesheet" href="css/color/color7.css"> -->
		<!--<link rel="stylesheet" href="css/color/color1.css">-->
		<!--<link rel="stylesheet" href="css/color/color2.css">-->
		<!--<link rel="stylesheet" href="css/color/color3.css">-->
		<!--<link rel="stylesheet" href="css/color/color4.css">-->
		<!--<link rel="stylesheet" href="css/color/color5.css">-->
		<!--<link rel="stylesheet" href="css/color/color6.css">-->
		<!--<link rel="stylesheet" href="css/color/color8.css">-->
		<link rel="stylesheet" href="css/color/color9.css">

		<link rel="stylesheet" href="#" id="colors">
		
    </head>
    <body>
	
<!-- 		<div class="color-plate ">
			<a class="icon"><i class="fa fa-cog fa-spin"></i></a>
			<h2>Ruby Color</h2>
			<div class="color-head">
				<span class="color1"></span>
				<span class="color2"></span>
				<span class="color3"></span>
				<span class="color4"></span>
				<span class="color5"></span>
				<span class="color6"></span>
				<span class="color7"></span>
				<span class="color8"></span>
				<span class="color9"></span>
			</div>
		</div> -->
		
		<!-- Header Area -->
		<header id="header" class="header">
			<div class="header-inner">
				<div class="container">
					<div class="row">
						<div class="col-md-3 col-sm-2 col-xs-2">
							<div class="logo">
								<a href="index.html"><img src="img/logo.png" alt="logo.png" height="45"></a>
							</div>
						</div>
						<div class="col-md-9 col-sm-10">
							<div class="mobile-menu"></div>
							<nav class="navbar navbar-default">
								<div class="collapse navbar-collapse">
									<ul id="nav" class="nav navbar-nav">
										<li class="current"><a href="#slider">Home</a></li>
										<li><a href="#features">Research</a></li>
										<!-- <li><a href="#Services">News</a></li> -->
										<li><a href="#latest-works">Publications</a></li>
										<li><a href="#blog">Members</a></li>
										<li><a href="#contact">Join us</a></li>
									</ul>
								</div>
							</nav>
						</div>
					</div>
				</div>
			</div>
		</header>
		<!--/End Header Area -->
		
		<!-- Slider Area -->
		<section id="slider">
				<div class="single-slider" style="background-image:url('img/background.jpg')">
					<div id="particles-js"></div>
					<div class="container">
						<div class="row">
							<div class="col-md-8 col-md-push-2">
								<div class="text">
									<h1 class="cd-headline clip is-full-width">
										OPEC
										<span class="cd-words-wrapper">
											<b class="is-visible">Photon</b>
											<b>Quantum</b>
											<b>Mathematics</b>
											<b>AI</b>
											<b>HPC</b>
										</span>
									</h1>
									<p style="font-size: 27px; color: yellow; white-space: nowrap; background-color: black; width: max-content; text-align: center;">Welcome to OPEC Lab at Australian National University & University of Otago.</p>
								</div>
							</div>
						</div>
					</div>					
				</div>
		</section>
		<!--/End Slider Area -->
		
		<!-- Research Area -->
		<section id="features" class="section">
			<div class="container" >
				<div class="row">
					<div class="col-md-12">
						<div class="section-title" style="text-align: center; width: 1170px; padding: 50px;">
							<h2>Lab Mission</h2>
							<!-- <p style="font-size: 21px; text-align: justify;">We aim to leverage silicon photonics technology to address the significant challenges faced by current computing architectures at both the technological and application levels. On the technological front, as Moore's Law reaches its limits, the performance gains of traditional semiconductor devices are slowing down, with increasing power consumption and heat dissipation issues further constraining overall system performance across all platforms, including mobile devices, embedded systems, laptops, servers, and data centers. On the application front, we are entering a new computing era that necessitates a shift from an algorithm-based computing paradigm to a learning-based, data-intensive paradigm, where human capabilities are scaled and augmented. To meet the growing computational demands and overcome power density limitations, the computing industry has embraced parallelism as the only viable approach to improving computational performance. Today, computing systems are being designed with tens to hundreds of computing cores integrated into a single chip, and hundreds to thousands of computing servers based on these chips are interconnected in data centers and supercomputers. However, power consumption remains a significant design challenge, and these highly parallel systems continue to face major challenges in energy efficiency, performance, and reliability.
							</p> -->
							<p style="font-size: 21px; text-align: justify;" style="width: 1300px; margin: 0;">We aim to use silicon photonics to overcome the limitations of current computing architectures, which are constrained by the slowing performance of traditional semiconductors and the shift towards data-intensive, learning-based paradigms, while addressing challenges in power consumption, efficiency, and reliability in highly parallel systems.
							</p>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-md-6 col-sm-12 col-xs-12 wow fadeInLeft" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<video controls width="100%" height="auto">
							<source src="img/singlecell.mp4" type="video/mp4">
							Your browser does not support the video tag.
						</video>
					</div>
					<div class="col-md-6 col-sm-12 col-xs-12 wow fadeInRight" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="feature-right">
							<div class="single-right">
								<i class="fa fa-cubes" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Future photonic accelerators</h3>
									<P>Designing reconfigurable heterogeneous photonic accelerator architectures using emerging photonic computing technologies.</P>
								</div>
							</div>
							<div class="single-right">
								<i class="fa fa-bar-chart" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Emerging interconnect technologies</h3>
									<P>Emerging interconnect technologies (photonic, wireless, RF, hybrid) for Network-on-Chips (NoCs) and embedded systems.</P>
								</div>
							</div>
							<div class="single-right">
								<i class="fa fa-magic" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Advanced parallel computing models</h3>
									<P>Future parallel computing models and architectures including Convolutional Neural Networks (CNNs), Deep Neural Networks (DNNs), near-data computing, and approximate computing.</P>
								</div>
							</div>
							<div class="single-right">
								<i class="fa fa-paper-plane-o" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Cloud computing and edge computing</h3>
									<P>Design a heterogeneous model combining cloud computing and edge computing to meet the needs of applications such as autonomous driving, real-time video processing, and intelligent manufacturing.
									</P>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		<!--/End Research Area -->
		
		<!-- News Area -->
		<section id="Services" class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>News</h2>
							</br>
						</div>
					</div>
				</div>
			</div>
		<section id="testimonials" class="section" data-stellar-background-ratio="0.5">
			<section class="bg">
				<div class="container">
					<div class="row">
						<div class="col-md-12 col-sm-12 col-xs-12">
							<div class="owl-carousel testimonial-slider">
								<div class="single-testimonial">
									<h2>scCrab has been accepted by Interdisciplinary Sciences: Computational Life Sciences</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for reference-guided cancer cell identification has been accepted for publication in Interdisciplinary Sciences: Computational Life Sciences. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>August 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scPRAM has been accepted by Bioinformatics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for predicting single-cell gene expression perturbation responses has been accepted for publication in Bioinformatics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>April 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>OpenAnnotateApi has been accepted by Bioinformatics Advances</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for efficiently annotating and analyzing chromatin accessibility of genomic regions has been accepted for publication in Bioinformatics Advances. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>April 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>EpiCarousel has been accepted by Bioinformatics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for memory- and time-efficient identification of metacells in atlas-level single-cell chromatin accessibility data has been accepted for publication in Bioinformatics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>April 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scButterfly has been accepted by Nature Communications</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for versatile single-cell cross-modality translation has been accepted for publication in Nature Communications. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>March 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>CASTLE has been accepted by Nature Computational Science</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for discrete and interpretable embedding of single-cell chromatin accessibility data has been accepted for publication in Nature Computational Science. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>March 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>CASCADE has been accepted by IEEE/ACM Transactions on Computational Biology and Bioinformatics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for annotating differentiating and imbalanced cell types in single-cell chromatin accessibility data has been accepted for publication in IEEE/ACM Transactions on Computational Biology and Bioinformatics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>February 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scCASE has been accepted by Nature Communications</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for enhancing single-cell chromatin accessibility data has been accepted for publication in Nature Communications. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>February 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>SCREEN has been accepted by Frontiers of Computer Science</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for predicting single-cell gene expression perturbation responses has been accepted for publication in Frontiers of Computer Science. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>January 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>Cofea has been accepted by Briefings in Bioinformatics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for feature selection in single-cell chromatin accessibility data has been accepted for publication in Briefings in Bioinformatics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>November 2023</span>
								</div>
								<div class="single-testimonial">
									<h2>Sijie won National Scholarship</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Sijie won the National Scholarship from Ministry of Education of China. National Scholarship is one of the most outstanding scholarships for graduates and undergraduates in China. Congratulations to Sijie!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>October 2023</span>
								</div>
								<div class="single-testimonial">
									<h2>scIBD has been accepted by Genome Biology</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for boosting the detection of heterotypic doublets in single-cell chromatin accessibility data has been accepted for publication in Genome Biology. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>September 2023</span>
								</div>
								<div class="single-testimonial">
									<h2>scEpiTools has been accepted by Journal of Genetics and Genomics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for comprehensively interrogating single-cell epigenomic tools has been accepted for publication in Journal of Genetics and Genomics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>September 2023</span>
								</div>
								<div class="single-testimonial">
									<h2>PAST has been accepted by Genome Research</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for latent feature extraction in spatial transcriptomics has been accepted for publication in Genome Research. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>September 2023</span>
								</div>
								
							</div>
						</div>
					</div>

				</div>
			</section>
		</section>
		</section>
		<!--/End News Area -->
		
		<!-- Publications Area -->
		<section id="latest-works" class="latest-works section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Publications</h2>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-md-12 col-sm-12 col-xs-12">
						<div class="works-menu wow fadeInUp" data-wow-duration="0.8s" data-wow-delay="0.3s">
							<ul>
								<li class="active" data-filter="*">All</li>
								<li data-filter=".preprints">Preprints</li>
								<li data-filter=".paper2024">2024</li>
								<li data-filter=".paper2023">2023</li>
								<li data-filter=".paper2022">2022</li>
								<li data-filter=".paper2021">2017-2021</li>
							</ul>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="isotop-active">
						<div class="col-md-12 col-sm-12 col-xs-12 paper2021 paper2022 paper2023 paper2024 preprints">
							<p>† first authors with equal contribution, * corresponding authors</p>
						</div>
						<div class="col-md-12 col-sm-12 col-xs-12 preprints">
							<p><i class="fa fa-paper-plane" aria-hidden="true"></i> <u>Sicheng Feng</u>, <u>Siyu Li</u>, Luonan Chen*, <u>Shengquan Chen*</u>. Unveiling potential threats: backdoor attacks in single-cell pretrained models.</p>
							<p><i class="fa fa-paper-plane" aria-hidden="true"></i> <u>Xuejian Cui</u>, Qijin Yin, <u>Zijing Gao</u>, <u>Zhen Li</u>, <u>Xiaoyang Chen</u>, <u>Shengquan Chen</u>, Wanwen Zeng, Rui Jiang. CREATE: cell-type-specific cis-regulatory elements identification via discrete embedding.</p>
							<p><i class="fa fa-paper-plane" aria-hidden="true"></i> Xiaofeng Wang†, Ruike Song†, Jiahao Zhang, <u>Shengquan Chen*</u>, Jianyu Zhou*. GATDE: A graph attention network with diffusion-enhanced protein-protein interaction for cancer classification.</p>
							<p><i class="fa fa-paper-plane" aria-hidden="true"></i> <u>Zhen Li</u>, <u>Xuejian Cui</u>, <u>Xiaoyang Chen</u>, <u>Zijing Gao</u>, Yuyao Liu, <u>Yan Pan</u>, <u>Shengquan Chen</u>, Rui Jiang*. Cross-modality representation and multi-sample integration of spatially resolved omics data.</p>
							<p><i class="fa fa-paper-plane" aria-hidden="true"></i> <u>Yuxi Li</u>, Yi Liu, Yuekang Li, Ling Shi, Gelei Deng, <u>Shengquan Chen</u>, Kailong Wang. Lockpicking LLMs: a logit-based jailbreak using token-level manipulation.</p>
							<p><i class="fa fa-paper-plane" aria-hidden="true"></i> Qiuchen Meng†, Xinze Wu†, Chen Li, Jiaqi Li, Xi Xi, Sijie Chen, <u>Shengquan Chen</u>, Jiaqi Li, Xiaowo Wang, Rui Jiang, Lei Wei*, Xuegong Zhang*. The full set of potential open regions (PORs) in the human genome defined by consensus peaks of ATAC-seq data.</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2024">
							<p>[31] <u>Heyang Hua†</u>, <u>Wenxin Long†</u>, <u>Yan Pan†</u>, <u>Siyu Li</u>, Jianyu Zhou*, Haixin Wang*, <u>Shengquan Chen*</u>. scCrab: a reference-guided ensemble method for cancer cell identification via Bayesian neural networks. <b>Interdisciplinary Sciences: Computational Life Sciences</b>, 2024, Accepted. [<a href="#" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scCrab" target="_blank">code</a>]</p>
							<p>[30] Zhihan Ruan, Weijun Zhou, Hong Liu, Jinmao Wei, Yichen Pan, Chaoyang Yan, Xiaoyi Wei, Wenting Xiang, Chengwei Yan, <u>Shengquan Chen</u>, Jian Liu*. Precise detection of cell-type-specific domains in spatial transcriptomics. <b>Cell Reports Methods</b>, 2024, 4(8):100841. [<a href="https://doi.org/10.1016/j.crmeth.2024.100841" target="_blank">link</a>, <a href="https://github.com/lyotvincent/Despot" target="_blank">code</a>]</p>
							<p>[29] <u>Qun Jiang†</u>, <u>Shengquan Chen†</u>, <u>Xiaoyang Chen</u>, Rui Jiang*. scPRAM accurately predicts single-cell gene expression perturbation response based on attention mechanism. <b>Bioinformatics</b>, 2024, btae265. [<a href="https://doi.org/10.1093/bioinformatics/btae265" target="_blank">link</a>, <a href="https://github.com/jiang-q19/scPRAM" target="_blank">code</a>]</p>
							<p>[28] <u>Zijing Gao</u>, Rui Jiang, <u>Shengquan Chen*</u>. OpenAnnotateApi: Python and R packages to efficiently annotate and analyze chromatin accessibility of genomic regions. <b>Bioinformatics Advances</b>, 2024, vbae055. [<a href="https://doi.org/10.1093/bioadv/vbae055" target="_blank">link</a>, <a href="https://github.com/ZjGaothu/OpenAnnotatePy" target="_blank">code</a>]</p>
							<p>[27] <u>Sijie Li</u>, <u>Yuxi Li</u>, Yu Sun, Yaru Li, <u>Xiaoyang Chen</u>, <u>Songming Tang</u>, <u>Shengquan Chen*</u>. EpiCarousel: memory- and time-efficient identification of metacells for atlas-level single-cell chromatin accessibility data. <b>Bioinformatics</b>, 2024, btae191. [<a href="https://doi.org/10.1093/bioinformatics/btae191" target="_blank">link</a>, <a href="https://github.com/biox-nku/epicarousel" target="_blank">code</a>]</p>
							<p>[26] <u>Yichuan Cao</u>, <u>Xiamiao Zhao</u>, <u>Songming Tang</u>, <u>Qun Jiang</u>, <u>Sijie Li</u>, <u>Siyu Li</u>, <u>Shengquan Chen*</u>. scButterfly: a versatile single-cell cross-modality translation method via dual-aligned variational autoencoders. <b>Nature Communications</b>, 2024, 15:2973. <b>Editors' Highlights</b>. [<a href="https://www.nature.com/articles/s41467-024-47418-x" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scButterfly" target="_blank">code</a>]</p>
							<p>[25] <u>Xuejian Cui</u>, <u>Xiaoyang Chen</u>, <u>Zhen Li</u>, <u>Zijing Gao</u>, <u>Shengquan Chen*</u>, Rui Jiang*. Discrete latent embedding of single-cell chromatin accessibility sequencing data for uncovering cell heterogeneity. <b>Nature Computational Science</b>, 2024, 4:346-359. [<a href="https://doi.org/10.1038/s43588-024-00625-4" target="_blank">link</a>, <a href="https://github.com/cuixj19/CASTLE" target="_blank">code</a>]</p>
							<p>[24] <u>Yuhang Jia†</u>, <u>Siyu Li†</u>, Rui Jiang, <u>Shengquan Chen*</u>. Accurate annotation for differentiating and imbalanced cell types in single-cell chromatin accessibility data. <b>IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2024, 21(3):461-471. [<a href="https://ieeexplore.ieee.org/document/10460166" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/CASCADE" target="_blank">code</a>]</p>
							<p>[23] <u>Songming Tang</u>, <u>Xuejian Cui</u>, Rongxiang Wang, <u>Sijie Li</u>, <u>Siyu Li</u>, Xin Huang, <u>Shengquan Chen*</u>. scCASE: accurate and interpretable enhancement for single-cell chromatin accessibility sequencing data. <b>Nature Communications</b>, 2024, 15:1629. <b>Editors' Highlights</b>. [<a href="https://www.nature.com/articles/s41467-024-46045-w" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scCASE" target="_blank">code</a>]</p>
							<p>[22] Haixin Wang†, <u>Yunhan Wang†</u>, <u>Qun Jiang†</u>, Yan Zhang*, <u>Shengquan Chen*</u>. SCREEN: predicting single-cell gene expression perturbation responses via optimal transport. <b>Frontiers of Computer Science</b>, 2024, Online. [<a href="https://journal.hep.com.cn/fcs/EN/10.1007/s11704-024-1014-2" target="_blank">link</a>, <a href="https://github.com/Califorya/SCREEN" target="_blank">code</a>]</p>
							<p>[21] Keyi Li†, <u>Xiaoyang Chen†</u>, Shuang Song, Lin Hou, <u>Shengquan Chen*</u>, Rui Jiang*. Cofea: correlation-based feature selection for single-cell chromatin accessibility data. <b>Briefings in Bioinformatics</b>, 2024, bbad458. [<a href="https://doi.org/10.1093/bib/bbad458" target="_blank">link</a>, <a href="https://github.com/likeyi19/Cofea" target="_blank">code</a>]</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2023">
							<p>[20] <u>Wenhao Zhang</u>, Rui Jiang, <u>Shengquan Chen*</u>, Ying Wang*. scIBD: a self-supervised iterative-optimizing model for boosting the detection of heterotypic doublets in single-cell chromatin accessibility data. <b>Genome Biology</b>, 2023, 24:225. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03072-y" target="_blank">link</a>, <a href="https://github.com/Ying-Lab/scIBD" target="_blank">code</a>]</p>
							<p>[19] <u>Zijing Gao†</u>, <u>Xiaoyang Chen†</u>, <u>Zhen Li†</u>, <u>Xuejian Cui†</u>, <u>Qun Jiang</u>, Keyi Li, <u>Shengquan Chen*</u>, Rui Jiang*. scEpiTools: a database to comprehensively interrogate analytic tools for single-cell epigenomic data. <b>Journal of Genetics and Genomics</b>, 2023, 51(4):462-465. [<a href="https://doi.org/10.1016/j.jgg.2023.09.011" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/scepitools/" target="_blank">tool</a>]</p>
							<p>[18] <u>Zhen Li</u>, <u>Xiaoyang Chen</u>, Xuegong Zhang, Rui Jiang, <u>Shengquan Chen*</u>. Latent feature extraction with a Prior-based self-Attention framework for Spatial Transcriptomics. <b>Genome Research</b>, 2023, Online. [<a href="https://genome.cshlp.org/content/early/2023/10/30/gr.277891.123" target="_blank">link</a>, <a href="https://github.com/lizhen18THU/PAST" target="_blank">code</a>]</p>
							<p>[17] Chen Li†, <u>Xiaoyang Chen†</u>, <u>Shengquan Chen</u>,  Rui Jiang*, Xuegong Zhang*. simCAS: an embedding-based method for simulating single-cell chromatin accessibility sequencing data. <b>Bioinformatics</b>, 2023, btad453. [<a href="https://doi.org/10.1093/bioinformatics/btad453" target="_blank">link</a>, <a href="https://github.com/Chen-Li-17/simCAS" target="_blank">code</a>]</p>
							<p>[16] <u>Siyu Li</u>, <u>Songming Tang</u>, <u>Yunchang Wang</u>, <u>Sijie Li</u>, <u>Yuhang Jia</u>, <u>Shengquan Chen*</u>. Accurate cell type annotation for single-cell chromatin accessibility data via contrastive learning and reference guidance. <b>Quantitative Biology</b>, 2023, 12(1):85-99. [<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/qub2.33" target="_blank">link</a>, <a href="https://github.com/biox-nku/rainbow" target="_blank">code</a>]</p>
							<p>[15] Rui Jiang, <u>Zhen Li</u>, <u>Yuhang Jia</u>, <u>Siyu Li</u>, <u>Shengquan Chen*</u>. SINFONIA: scalable identification of spatially variable genes for deciphering spatial domains. <b>Cells</b>, 2023, 12(4):604. [<a href="https://doi.org/10.3390/cells12040604" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/SINFONIA" target="_blank">code</a>]</p>
						</div>
						
						<div class="col-md-12 col-sm-12 col-xs-12 paper2022">
							<p>[14] <u>Shengquan Chen*</u>, Rongxiang Wang, <u>Wenxin Long</u>, Rui Jiang*. ASTER: accurately estimating the number of cell types in single-cell chromatin accessibility data. <b>Bioinformatics</b>, 2022, btac842. [<a href="https://doi.org/10.1093/bioinformatics/btac842" target="_blank">link</a>, <a href="https://github.com/biox-nku/aster" target="_blank">code</a>]</p>
							<p>[13] Zheng Zhang, <u>Shengquan Chen</u>, Zhixiang Lin*. RefTM: reference-guided topic modeling of single-cell chromatin accessibility data. <b>Briefings in Bioinformatics</b>, 2022, bbac540. [<a href="https://doi.org/10.1093/bib/bbac540" target="_blank">link</a>, <a href="https://github.com/cuhklinlab/RefTM" target="_blank">code</a>]</p>
							<p>[12] <u>Xiaoyang Chen†</u>, <u>Shengquan Chen†</u>, Shuang Song, <u>Zijing Gao</u>, Lin Hou, Xuegong Zhang, Hairong Lv, Rui Jiang*. Cell type annotation of single-cell chromatin accessibility data via supervised Bayesian embedding. <b>Nature Machine Intelligence</b>, 2022, 4:116-126. [<a href="https://doi.org/10.1038/s42256-021-00432-w" target="_blank">link</a>, <a href="https://github.com/xy-chen16/EpiAnno" target="_blank">code</a>]</p>
							<p>[11] Xi Xi, Haochen Li, <u>Shengquan Chen</u>, Tingting Lv, Tianxing Ma, Rui Jiang, Ping Zhang, Wing Hung Wong*, Xuegong Zhang*. Unfolding the genotype-to-phenotype black box of cardiovascular diseases through cross-scale modeling. <b>iScience</b>, 2022, 25(8):104790. [<a href="https://www.sciencedirect.com/science/article/pii/S2589004222010628" target="_blank">link</a>, <a href="https://github.com/xixi-cathy/GRPath" target="_blank">code</a>]</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2021">
							<p>[10] <u>Shengquan Chen†</u>, Guanao Yan†, Wenyu Zhang, Jinzhao Li, Rui Jiang*, Zhixiang Lin*. RA3 is a reference-guided approach for epigenetic characterization of single cells. <b>Nature Communications</b>, 2021, 12:2177. [<a href="https://www.nature.com/articles/s41467-021-22495-4" target="_blank">link</a>, <a href="https://github.com/cuhklinlab/RA3" target="_blank">code</a>]</p>
							<p>[9] <u>Shengquan Chen</u>, Qiao Liu, <u>Xuejian Cui</u>, Zhanying Feng, Chunquan Li, Xiaowo Wang, Xuegong Zhang, Yong Wang, Rui Jiang*. OpenAnnotate: a web server to annotate the chromatin accessibility of genomic regions. <b>Nucleic Acids Research</b>, 2021, 49(W1):W483-W490. [<a href="https://doi.org/10.1093/nar/gkab337" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/openannotate/" target="_blank">tool</a>]</p>
							<p>[8] Wanwen Zeng†, <u>Shengquan Chen†</u>, <u>Xuejian Cui†</u>, <u>Xiaoyang Chen</u>, <u>Zijing Gao</u>, Rui Jiang*. SilencerDB: a comprehensive database of silencers. <b>Nucleic Acids Research</b>, 2021, 49(D1):D221-D228. [<a href="https://academic.oup.com/nar/article-abstract/49/D1/D221/5921294" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/silencerdb/" target="_blank">tool</a>]</p>
							<p>[7] <u>Shengquan Chen</u>, Mingxin Gan, Hairong Lv, Rui Jiang*. DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers. <b>Genomics, Proteomics & Bioinformatics</b>, 2021, 19(4):565-577. [<a href="https://www.sciencedirect.com/science/article/pii/S1672022921000127" target="_blank">link</a>, <a href="https://github.com/ShengquanChen/DeepCAPE" target="_blank">code</a>]</p>
							<p>[6] <u>Shengquan Chen</u>, Boheng Zhang, <u>Xiaoyang Chen</u>, Xuegong Zhang, Rui Jiang*. stPlus: a reference-based method for the accurate enhancement of spatial transcriptomics. <b>ISMB/ECCB 2021, Bioinformatics</b>, 2021, 37(S1):i299-i307. [<a href="https://doi.org/10.1093/bioinformatics/btab298" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/software/stPlus/" target="_blank">tool</a>]</p>
							<p>[5] Qiao Liu, <u>Shengquan Chen</u>, Rui Jiang*, Wing Hong Wong*. Simultaneous deep generative modeling and clustering of single cell genomic data. <b>Nature Machine Intelligence</b>, 2021, 3:536-544. [<a href="https://www.nature.com/articles/s42256-021-00333-y" target="_blank">link</a>, <a href="https://github.com/kimmo1019/scDEC" target="_blank">code</a>]</p>
							<p>[4] <u>Xiaoyang Chen</u>, <u>Shengquan Chen</u>, Rui Jiang*. EnClaSC: A novel ensemble approach for accurate and robust cell-type classification of single-cell transcriptomes. <b>BMC Bioinformatics</b>, 2020, 21(13):1-16. [<a href="https://link.springer.com/article/10.1186/s12859-020-03679-z" target="_blank">link</a>, <a href="https://github.com/xy-chen16/EnClaSC" target="_blank">code</a>]</p>
							<p>[3] <u>Shengquan Chen</u>, Kui Hua, Hongfei Cui, Rui Jiang*. VPAC: Variational projection for accurate clustering of single-cell transcriptomic data. <b>BMC Bioinformatics</b>, 2019, 20(7):139-151. [<a href="https://link.springer.com/article/10.1186/s12859-019-2742-4" target="_blank">link</a>, <a href="https://github.com/ShengquanChen/VPAC" target="_blank">code</a>]</p>
							<p>[2] Shaoming Song, Hongfei Cui, <u>Shengquan Chen</u>, Qiao Liu, Rui Jiang*. EpiFIT: functional interpretation of transcription factors based on combination of sequence and epigenetic information. <b>Quantitative Biology</b>, 2019, 7(3):233-243. [<a href="https://link.springer.com/article/10.1007/s40484-019-0175-8" target="_blank">link</a>, <a href="https://health.tsinghua.edu.cn/openness/EpiFIT/" target="_blank">tool</a>]</p>
							<p>[1] Xu Min†, Wanwen Zeng†, <u>Shengquan Chen</u>, Ning Chen, Ting Chen, Rui Jiang*. Predicting enhancers with deep convolutional neural networks. <b>BMC Bioinformatics</b>, 2017, 18(13):478. [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1878-3" target="_blank">link</a>, <a href="https://github.com/minxueric/DeepEnhancer" target="_blank">code</a>]</p>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-md-12">
						<div class="button-head">
							<div class="button">
								<span></span>
								<a href="https://scholar.google.com/citations?user=z5p5bjIAAAAJ&hl=en" class="btn" target="_blank">Load More</a>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		<!--/End Publications Area -->
		
		<!-- Members -->
		<section id="blog" class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Members</h2>
						</div>
					</div>
				</div>
				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/amanda.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Principal Investigator</h3>
								<h2>Amanda Barnard AM</h2>
								<p style="text-align:center;">School of Computing, ANU</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Senior Professor. (2020.1-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D., RMIT U. (2001-2003)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Eng., Xiamen U. (1998-2001)</p> -->
								<a href="https://comp.anu.edu.au/people/amanda-barnard/" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/zhiyihuang.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Professor</h3>
								<h2>Zhiyi Huang</h2>
								<p style="text-align:center;">School of Computing, UO</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Professor. (1999.6-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D., NUDT. (1988-1992)</p>
								<!-- <p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2021.9-2024.7)</p> -->
								<a href="https://www.otago.ac.nz/school-of-computing/our-people/zhiyi_huang" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/haibo.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Associate Professor</h3>
								<h2>Haibo Zhang</h2>
								<p style="text-align:center;">School of Computing, ANU&UO</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Associate Professor. (2011.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D., U of Adelaide.  (2006-2008)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Eng., Tsinghua U. (2018.9-2022.7)</p> -->
								<a href="https://researchers.anu.edu.au/researchers/zhang-hxx" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/wendy.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Senior Lecturer</h3>
								<h2>Wendy Chen</h2>
								<p style="text-align:center;">School of Systems & Computing, UNSW</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Senior Lecturer. (2024.2-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D., U of Adelaide. (2006-2009)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>M.Phil., XJTU. (2020.9-2022.7)</p> -->
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., ZZU. (2016.9-2020.7)</p> -->
								<a href="https://www.unsw.edu.au/staff/wendy-chen" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/yaoyufei.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Joint Ph.D. Student with Australian National Univ</h3>
								<h2>Hao</h2>
								<p style="text-align:center;">School of Computing</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D. Student, UO. (2021.4-)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>M.Phil., XJTU. (2021.9-2023.7)</p> -->
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., NWAFU. (2016.9-2020.7)</p> -->
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/lisijie.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Joint Ph.D. Student with Australian National Univ</h3>
								<h2>Chengpeng</h2>
								<p style="text-align:center;">School of Computing, UO</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D. Student, UO. (2021.9-)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Nankai U. (2018.9-2022.7)</p> -->
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/gaofei.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Ph.D. Student</h3>
								<h2>Fei Gao</h2>
								<p style="text-align:center;">School of Computing</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D. Student, UO. (2022.4-)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Eng., Tsinghua U. (2018.9-2022.7)</p> -->
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/yuxiaojie.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Ph.D. Student</h3>
								<h2>Xiaojie Yu</h2>
								<p style="text-align:center;">School of Computing</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D. Student, UO. (2022.4-)</p>
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., SCUT. (2019.9-2023.7)</p> -->
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/zhangqian.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Ph.D. Student</h3>
								<h2>Qian Zhang</h2>
								<p style="text-align:center;">School of Computing</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D. Student, UO. (2024.4-)</p>
								<!-- <p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, Shenzhen Institute of Advanced Technology, CAS. (2023.10-)</p> -->
								<!-- <p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Nankai U. (2020.9-2024.7)</p> -->
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<!-- <div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/lisiyu.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Siyu Li</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2024.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, Department of Statistics, The Chinese University of Hong Kong. (2024.1-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Nankai U. (2020.9-2024.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/liyongkang.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Yongkang Li</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2024.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Nankai U. (2020.9-2024.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/fengsicheng.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">B.Eng. Student</h3>
								<h2>Sicheng Feng</h2>
								<p style="text-align:center;">College of Computer Science</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Eng., Nankai U. (2021.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, School of Engineering, Westlake University (2024.7-)</p>
								<p>Next position: M.Phil. Student, School of Mathematical Sciences, Nankai University</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/xiangshihan.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">B.Sc. Student</h3>
								<h2>Shihan Xiang</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2021.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/lianghaitian.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">B.Sc. Student</h3>
								<h2>Haitian Liang</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2023.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/linyiming.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">B.Sc. Student</h3>
								<h2>Yiming Lin</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2023.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/kaikai.png" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Encourager</h3>
								<h2>Kaikai</h2>
								<p style="text-align:center;">School of Happiness</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Home U. (2022.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section> -->
		<!-- /End Members Area -->
		
		<!-- Alumni Area -->
		<!-- <section id="alumni" class="latest-works section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Alumni</h2>
						</div>
					</div>
				</div>
				<div class="row" style="margin-top: 20px;">
					<div class="col-md-12 col-sm-12 col-xs-12">
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Travis Dai, Ph.D., 2022.7-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Academy of Mathematics and Systems Science, Chinese Academy of Sciences</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Xiamiao Zhao, Undergraduate student, 2022.7-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in School of Mathematical Sciences, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yunhan Wang, Undergraduate student, 2022.11-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Statistics, Renmin University of China</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yuxi Li, Undergraduate student, 2022.7-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Cyber Science and Engineering, Huazhong University of Science and Technology</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Wenhao Zhang, Postgraduate student, 2022.7-2023.10</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in School of Aerospace Engineering, Xiamen University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Xiaoyang Chen, Postgraduate student, 2022.2-2024.1</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Zijing Gao, Postgraduate student, 2023.1-2024.4</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Xuejian Cui, Postgraduate student, 2022.8-2024.5</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Qun Jiang, Postgraduate student, 2023.5-2024.5</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yuhang Jia, Undergraduate student, 2022.4-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Mathematical Sciences, Nankai University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Siyu Li, Undergraduate student, 2022.4-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Mathematical Sciences, Nankai University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Wenxin Long, Undergraduate student, 2022.5-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Statistics, Eberly College of Science, Penn State University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Shuchen Zhu, Undergraduate student, 2022.10-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Academy for Advanced Interdisciplinary Studies, Peking University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yifan Xu, Undergraduate student, 2023.9-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in Financial Engineering, New York University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Hongyang Pang, Undergraduate student, 2023.12-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in Department of Statistics and Data Science, Northwestern University</p>
					</div>						
				</div>
			</div>
		</section> -->
		<!--/End Alumni Area -->

		<!--/ Contact Area -->
		<section id="contact" class="contact section" data-stellar-background-ratio="0.5">
			<div class="container">
				<div class="row">
					<div class="contact-head">
						<div class="col-md-4 col-sm-4 col-xs-12" style="width: 50%; padding: 0; margin: 0;">
							<div class="contact-left" style="width: 100%; height: 100%; padding: 0; margin: 0;">
								<!-- Map container without any padding or margins -->
								<script type="text/javascript" id="clustrmaps" src="//clustrmaps.com/map_v2.js?d=ZmAg-gzvwTi5XRa7_HUhKfHGDox-Pzodk4Tc4qzrcQ0&cl=ffffff&w=a" async></script>
							</div>
						</div>
						<div class="col-md-8 col-sm-8 col-xs-12" style="width: 50%;">
							<div class="contact-right" style="display: flex; justify-content: space-between;">
								<div style="width: 48%;">
									<h3>Contact information</h3>
									<p class="p1">Hao Zhang</p>
									<p>Hao[dot]Zhang[at]gmail[dot]com</p>
									<p>Room 423, Department of Informatics Theory and Data Science</p>
									<p>School of Mathematical Sciences, Nankai University, Tianjin, China</p>
								</div>
								<div style="width: 48%;">
									<h3>Contact information</h3>
									<p class="p1">Chengpeng Xia</p>
									<p>Chengpeng[dot]Xia[at]gmail[dot]com</p>
									<p>Room 423, Department of Informatics Theory and Data Science</p>
									<p>School of Mathematical Sciences, Nankai University, Tianjin, China</p>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		
		
		
		<!--/End Contact Area -->
		
		<!-- Footer Area -->
		<footer id="footer" class="footer section">
			<div class="copyright">
				<div class="container">
					<div class="row">
						<div class="col-md-12 col-sm-12 col-xs-12">
							<div class="copyright-content">
								<p>Copyright &copy; 2024. OPEC at ANU-UO. All rights reserved.</p>
							</div>
						</div>
					</div>
				</div>
			</div>
		</footer>



		<!--/End Footer Area -->
			
        <script src="js/jquery.min.js"></script>
        <script src="js/bootstrap.min.js"></script>
		<script src="js/colors.js"></script>
        <script src="js/jquery.nav.js"></script>
        <script src="js/jquery.scrollUp.min.js"></script>
        <script src="js/jquery.slicknav.min.js"></script>
		<script src="js/waypoints.min.js"></script>
        <script src="js/owl.carousel.min.js"></script>
		<script src="js/isotope.pkgd.min.js"></script>
		<script src="js/wow.min.js"></script>
		<script src="js/jquery.stellar.min.js"></script>
		<script src="js/jquery.magnific-popup.min.js"></script>
		<script src="js/animate-text.js"></script>
		<script src="js/particles.min.js"></script>
		<script src="js/particle-code.js"></script>
		<script src="js/jquery.counterup.min.js"></script>
		
        <script type="text/javascript" src="js/gmaps.min.js"></script>
        <script src="js/main.js"></script>
    </body>
</html>